domains/Discussions/Controllers/AnswersStoreController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Domains\Discussions\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Discussions\Models\Answer;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class AnswersStoreController extends Controller
+{
+    private Answer $answer;
+    private Question $question;
+
+    public function __construct(Answer $answer, Question $question)
+    {
+        $this->answer   = $answer;
+        $this->question = $question;
+    }
+
+    public function __invoke(int $questionId, Request $request): Response
+    {
+        $this->validate($request, [
+            'content' => ['required', 'string'],
+        ]);
+
+        $question = $this->question->findOrFail($questionId);
```

Comment:
We can use a validation rule for this: `exists`, but we should also check that the question isn't soft-deleted, too.